bikeshed/boilerplate/whatwg/defaults.include

```diff
@@ -5,7 +5,8 @@
   "Force Crossorigin": "yes",
   "Status": "LS",
   "No Editor": "yes",
-  "Boilerplate": "omit style-md-lists, omit style-autolinks, omit style-selflinks, omit style-counters, omit style-syntax-highlighting, omit feedback-header, omit conformance, omit issues-index",
+  "Boilerplate": "omit style-md-lists, omit style-autolinks, omit style-selflinks, omit style-counters, omit style-syntax-highlighting, omit feedback-header, omit conformance, omit issues-index, style-mdn-anno no, script-mdn-anno no",
```

Comment:
Should this be "omit" prefix instead of "no" suffix?

Reply:
Thanks — yeah, changed to "omit" for consistency